src_kt/java/arcs/testutil/BUILD

```diff
@@ -0,0 +1,12 @@
+package(default_visibility = ["//visibility:public"])
+
+load("//tools/build_defs/kotlin:rules.bzl", "kt_jvm_library")
+
+# The `testutil` library can be JVM-only, since our unit tests are run on the JVM.
+kt_jvm_library(
+    name = "testutil",
+    srcs = glob(["*.kt"]),
+    deps = [
+        "//third_party/java/junit",
+    ]
```

Comment:
Does `testonly = True` work with bazel? If so, might be worth adding

Reply:
Good idea. Done.